cfme/common/__init__.py

```diff
@@ -236,15 +236,22 @@ class SummaryTable(object):
 
     MULTIKEY_LOC = '../../../tbody/tr[1]/td/strong'
 
-    def __init__(self, o, text, entry):
+    def __init__(self, o, text, entry, skip_load=False):
         self._object = o
         self._text = text
         self._entry = entry
         self._raw_keys = []
         self._keys = []
-        self.load()
+        self._multitable = False
```

Comment:
In this case probably a console message with level WARN. 

Btw. next time you should ping me on IRC, I get so many GH notifications I no longer pay attention to all of them.